src/Components/Search/NewSearch/Mobile/SearchResultsList/InfiniteScrollSentinel.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box } from "@artsy/palette"
+import { useIntersectionObserver } from "Utils/Hooks/useIntersectionObserver"
+import { FC } from "react"
+
+interface InfiniteScrollSentinelProps {
+  onNext: () => void
+}
+
+export const InfiniteScrollSentinel: FC<InfiniteScrollSentinelProps> = ({
```

Comment:
**suggestion**: make it reusable. since we already have components with the same code and logic ([component 1](https://github.com/artsy/force/blob/145c983c61757b89ba1dc7c864d8da57e2d6035e/src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/ArtworksInfiniteScrollSentinel.tsx#L9), [component 2](https://github.com/artsy/force/blob/145c983c61757b89ba1dc7c864d8da57e2d6035e/src/Components/Notifications/NotificationsListScrollSentinel.tsx#L8), [component 3](https://github.com/artsy/force/blob/145c983c61757b89ba1dc7c864d8da57e2d6035e/src/Apps/Article/Components/ArticleInfiniteScroll.tsx#L92-L99))